src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useContext, useEffect } from "react";
 import { Redirect, Route, Switch, useHistory } from "react-router-dom";
-import './App.css';
+import 'sass/app.scss';
```

Comment:
Can you sync with @floraguolr about the work you've done so far with this and how you would envision her implementing style guide off of the work you've started here to migrate over to scss?